logstash-core/lib/logstash/patches/resolv.rb

```diff
@@ -0,0 +1,32 @@
+require "resolv"
+
+# ref: https://github.com/logstash-plugins/logstash-filter-dns/issues/40
+#
+# JRuby versions prior to 9.1.16.0 have a bug which crashes IP address
+# the resolver after 64k unique IP addresses resolutions.
+
+version = JRUBY_VERSION.split(".").map(&:to_i)
```

Comment:
let's do the same here as in https://github.com/logstash-plugins/logstash-filter-dns/pull/45

Reply:
done.